specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+        "id": {
+          "type": "string",
+          "description": "ARM Id of the storage task assignments, associated with the storage tasks.",
+          "readOnly": true
+        }
+      },
+      "description": "Fetch the Storage task assignment ARM ids."
+    },
+    "StorageTasksRunSummaryList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageTaskRunStatus"
+          },
+          "x-ms-identifiers": [],
+          "description": "Gets storage tasks run result summary."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Request URL that can be used to query next page of storage task run results summary. Returned when the number of run instances and summary reports exceed maximum page size.",
+          "readOnly": true
+        }
+      },
+      "description": "Fetch Storage Tasks Run Summary."
+    },
+    "StorageTaskRunStatus": {
+      "type": "object",
+      "properties": {
+        "taskAssignmentId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Represents the Storage Task Assignment Id associated with the storage task that provided an execution context."
+        },
+        "storageAccountId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Represents the Storage Account Id where the storage task definition was applied and executed."
+        },
+        "startTime": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Start time of the run instance. Filter options such startTime gt '2023-06-26T20:51:24.4494016Z' and other comparison operators can be used as described for DateTime properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "finishTime": {
+          "readOnly": true,
+          "type": "string",
+          "description": "End time of the run instance. Filter options such startTime gt '2023-06-26T20:51:24.4494016Z' and other comparison operators can be used as described for DateTime properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "objectsTargetedCount": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Total number of objects that meet the condition as defined in the storage task assignment execution context. Filter options such as objectsTargetedCount gt 50 and other comparison operators can be used as described for Numerical properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "objectsOperatedOnCount": {
+          "readOnly": true,
+          "type": "string",
```

Comment:
>string [](http://example.com/codeflow?start=19&length=6)

Type as number or int?

Reply:
No we need this to be a read-only string, so that it can be queried. it is a kind of limitation from the underlying table implementation.

Reply:
Okay